src/java.base/share/classes/java/io/FileInputStream.java

```diff
@@ -210,12 +209,7 @@ public FileInputStream(FileDescriptor fdObj) {
      * @param name the name of the file
      */
     private void open(String name) throws FileNotFoundException {
```

Comment:
If method such as this is private, and only delegates to the 0-suffixed native method, would't it be better to just call the 0-suffixed method directly?

Reply:
Historically these native methods were wrapped in order to support instrumentation, I didn't want to touch in this PR.

Reply:
And presumably all these 0-suffixed methods will eventually be replaced with FFM calls anyway.